src/Module/Api/Mastodon/Accounts/VerifyCredentials.php

```diff
@@ -52,6 +52,6 @@ protected function rawContent(array $request = [])
 
 		// @todo Support the source property,
 		$account = DI::mstdnAccount()->createFromContactId($cdata['user'], $uid);
-		System::jsonExit($account);
+		$this->response->exit('account', ['account' => $account->toArray()]);
```

Comment:
Mastodon endpoints only support a JSON output. So wouldn't it be better having just a `jsonExit` function in the `response` class?

Reply:
Currently, it's JSON per default, but I will enforce it per parameter

Reply:
What I meant was that we arbitrary add parameters here (see `'account'`) that are only used when not exporting via JSON. This is not elegant.